pkg/cache/informer_cache.go

```diff
@@ -140,6 +140,12 @@ func (ip *informerCache) GetInformer(obj runtime.Object) (Informer, error) {
 	return i.Informer, err
 }
 
+// NeedLeaderElection implements the LeaderElectionRunnable interface
```

Comment:
Unfortunately we can not verify the interface implementation via a `var _ manager.LeaderElectionRunnable = &informerCache{}` as that would result in an import cycle